tests/test_sse.py

```diff
@@ -76,16 +62,11 @@ async def func(request: web.Request) -> web.StreamResponse:
 
     # check that EventSourceResponse object works only
     # with GET method
-    resp = await session.request("POST", url)
+    resp = await client.post("/")
     assert 405 == resp.status
```

Comment:
Close should still happen on the response.
```suggestion
    async with await client.post("/") as resp:
        assert 405 == resp.status
```